extensions/amp-lightbox/1.0/component.js

```diff
@@ -70,6 +70,7 @@ function BentoLightboxWithRef(
         setVisible(true);
       },
       close: () => setVisible(false),
+      toggle: (visible) => setVisible(visible),
```

Comment:
It's fine that we don't compare state here as we did before, since `useState` should already prevent useless re-renders.